parcel/ABIFMparcel.jl

```diff
@@ -0,0 +1,230 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+
```

Comment:
I would prefer to also add this file in a separate PR with parcel model updates

Reply:
For now, I'm using this file as a test to try to reproduce the figure in Knopf & Alpert. Once I confirm that the ABIFM J function works correctly, I was going to delete it. Would it be better if I wait until the parcel model updates on the parcel branch are done before trying to do this?

Reply:
Which figure are you thinking of? I thought there was one that we wanted to reproduce that did not require the parcel model. 

But if it makes it easier in testing, then lets keep it for now